### PR TITLE
fixing multisig

### PR DESCRIPTION
**Pull request by mrsmkl**
## Description

Add a description of your changes here.

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

File: contracts/governance/NeverminedConfig.sol [hunk 1]
```diff
@@ -93,6 +93,14 @@ INVMConfig
         return marketplaceFee;
     }
 
+    function getMarketplaceFee2()
```
Comment (aaitor) [inline contracts/governance/NeverminedConfig.sol:96]: What's the meaning of this function?

File: contracts/templates/DIDSalesTemplate.sol [hunk 2]
```diff
@@ -37,6 +37,10 @@ contract DIDSalesTemplate is BaseEscrowTemplate {
     TransferDIDOwnershipCondition internal transferCondition;
     EscrowPaymentCondition internal rewardCondition;
 
+    function id() public returns (uint) {
```
Comment (aaitor) [inline contracts/templates/DIDSalesTemplate.sol:40]: what's the meaning of this `id`?

File: scripts/deploy/truffle-wrapper/deploy/initializeContracts.js [hunk 3]
```diff
@@ -96,7 +96,7 @@ async function initializeContracts({
         addressBook.NeverminedConfig = await zosCreate({
             contract: 'NeverminedConfig',
             ctx,
-            args: [roles.ownerWallet, roles.governorWallet],
+            args: [roles.ownerWallet, roles.deployer],
```
Comment (aaitor) [inline scripts/deploy/truffle-wrapper/deploy/initializeContracts.js:99]: I think is convenient to separate the `deployer` of the `governor` role. In a DAO setup both roles can be played by different entities.

Comment (aaitor) [reply scripts/deploy/truffle-wrapper/deploy/initializeContracts.js:99]: If that is the case, probably here what would be convenient is to use a multisig for the `governor`

Comment (mrsmkl) [reply scripts/deploy/truffle-wrapper/deploy/initializeContracts.js]: I forgot to add the transactions to change these at the end...

File: contracts/templates/DIDSalesTemplate.sol (hunk 2 above)
Comment (mrsmkl) [reply contracts/templates/DIDSalesTemplate.sol:41]: All contracts need to have different bytecodes, otherwise the OZ upgrades become messed up.

Comment (aaitor) [reply contracts/templates/DIDSalesTemplate.sol:41]: Do you think makes sense to make this homogeneous for all the templates?
Maybe adding a `TEMPLATE_TYPE` variable to each template that is returned
```
bytes32 private constant TEMPLATE_TYPE = keccak256('DIDSalesTemplate')
```

**Review by aaitor — APPROVED** [review]

**Pull request merged**